package.json

```diff
@@ -16,7 +16,7 @@
     "@mui/styled-engine-sc": "^5.8.0",
     "@mui/styles": "^5.8.7",
     "@mui/x-date-pickers": "^5.0.0-alpha.7",
-    "@terraware/web-components": "^1.0.5",
+    "@terraware/web-components": "^1.0.7",
```

Comment:
This won't currently build until I get the web-components PR checked in and published to npm.
I tested using the distribution build in web-components locally.